inception-doc/src/main/resources/META-INF/asciidoc/user-guide/workload.adoc

```diff
@@ -17,25 +17,60 @@
+done by adding "workload.dynamic.enabled=true" to the settings.properties file.
 
-Similar to the monitoring page, the workload page shall give project manager a fast overview of all documents
-and users within the current project.
+After activating the workload feature in the settings.properties file, a project manager has the possibility
+to enable the utilization of the dynamic workload feature in the new "Workload" tab in the projects settings.
+In the dropdown, the project manager can decide, whether the standard monitoring or the dynamic workload feature shall be used.
+There can only be one of them in use at any time. Still, switching between them is always possible.
+Upon selecting the dynamic workload from the dropdown and confirming it with the corresponding button, the
+project manager is able to select the new workload tab from the projects menu bar. Furthermore, from now on
+the overall annotation workflow will also change. An annotator will no more be able to choose which document he/she
+wants to annotate, but rather gets automatically a new document assigned upon finishing the current one. Therefore,
+a "switching" between documents, which have not been finished yet, is no more possible.
+
+image::workload_settings.png[align="center"]
+
+
+Similar to the monitoring page, the workload page gives the project manager a fast overview of all documents
+and users within the current project. Additionally, the automatic distribution of documents to annotators can be
+modified.
 
 image::workload.png[align="center"]
 
 Therefore, it mainly consists of a substantial, but easy to understand table containing the data a project manager
-needs from their documents.
-Overall, there are three different tabs which enable advanced features for the workload page.
-. Filters: Different filters can be applied here to make the table easier to read and get faster the required information needed.
-. Annotators: In this menu, a project manager is able to execute different tasks regarding the annotators of the project. For example,
+needs from their documents. This data is represented by a unique row for each individual document in the project. The following
+columns with the respective data are displayed:
+1. State: This represents the overall state of the document within the project. For further details on the available states please continue reading here. [..]
+2. Document: The document name.
+3. Assigned: Represented by a number. This number reflects the amount of annotators, who are working currently on the document.
+4. Finished: Also represented by a number. However, other than the "Assigned" column, this column represents the number of annotators, who have already finished their work on the document.
+5. Annotators: This column represents the name of all annotators, who have something to do with the respective document. Either they are working on it at the moment, or they have already finished their work on it.
```

Comment:
can be simplified to: "...the names of all annotators who are either working on the document or have already finished it."